### ISSUE TITLE
Copy/Paste Action Steps

### ISSUE THREAD

**Issue by MeestorX** [bitfocus/companion#2358]
### Is this a feature relevant to companion itself, and not a module?

- [X] I believe this to be a feature for companion, not a module

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the feature

Copy/Pasting of Action steps.
That is all.

### Usecases

Super handy.

**Comment by DanielBaulig** [comment]
I'd like to chip in here, because I've proposed a similar feature before.

v3 has the ability to clone and move actions around (on the page), which does 80% of the lifting of copy + paste. I think a critical UX part still missing that most copy + paste implementations would provide is an easy way to move an action from one button to another or from one step to another. This (appears) to currently not be possible and is what I think most people might be missing even with clone being available now.

**Comment by MeestorX** [comment]
I agree! I would love to have Cut/Copy/Paste functionality as much as possible everywhere. Drag/Drop in a browser (except with uploading files) is really hard to get to work well. 
I was a year or two into Companion before I realized I could Copy/Paste buttons on the SD without using the (odd-functioning) copy/move buttons!

**Comment by AJolly** [comment]
I'd love an OS level (clipboard) copy paste - ie - when you copy a button, it exports it to the system clipboard. It'd make it far easier to copy/paste buttons between different browser windows and different computers.
